tensorflow/lite/kernels/internal/reference/strided_slice.h

```diff
@@ -23,69 +23,44 @@ namespace tflite {
 namespace reference_ops {
 template <typename T>
 inline void StridedSlice(const tflite::StridedSliceParams& op_params,
-                         const RuntimeShape& unextended_input_shape,
-                         const T* input_data,
-                         const RuntimeShape& unextended_output_shape,
-                         T* output_data) {
-  using strided_slice::LoopCondition;
-  using strided_slice::StartForAxis;
-  using strided_slice::StopForAxis;
-  // Note that the output_shape is not used herein.
-  tflite::StridedSliceParams params_copy = op_params;
+						 const RuntimeShape& input_shape,
```

Comment:
use space instead of tab.

Reply:
@abattery : Thanks for review! I wonder whether this should be caught in clang-format ? Cause i am pretty sure i ran this when i raised the PR :).